app/controllers/katello/api/v2/content_uploads_controller.rb

```diff
@@ -10,7 +10,7 @@ class Api::V2::ContentUploadsController < Api::V2::ApiController
     param :repository_id, :number, :required => true, :desc => N_("repository id")
     param :size, :number, :required => true, :desc => N_("Size of file to upload")
     param :checksum, String, :required => false, :desc => N_("Checksum of file to upload")
-    param :content_type, RepositoryTypeManager.uploadable_content_types(false).map(&:label), :required => false, :desc => N_("content type ('deb', 'docker_manifest', 'file', 'ostree', 'rpm', 'srpm')")
+    param :content_type, RepositoryTypeManager.uploadable_content_types(false).map(&:label), :required => false, :desc => N_("content type ('deb', 'docker_manifest', 'file', 'ostree_ref', 'rpm', 'srpm')")
```

Comment:
What if we used `uploadable_content_types` to create the description? Then other generic content types would automatically show up here.

Reply:
you just can't do that within an N_() block for translation, granted these values don't need to be translated :)

Maybe lets just leave out the 'cotnent type' text and just list the possible values.

Reply:
@jjeffers i think we can remove this right?

Reply:
oops sorry ignore me ;)